backend/src/utils/migrations.py

```
@@ -1,6 +1,36 @@
+        with engine.connect() as conn:
+            conn.execute(text("DELETE FROM alembic_version"))
+            conn.commit()
+        engine.dispose()
+        logger.info("Cleared stale alembic_version table")
+
+        # Stamp to head after clearing
+        alembic_cfg = Config("alembic.ini")
+        command.stamp(alembic_cfg, "head")
+        logger.info("Database stamped to head")
+    except Exception as e:
+        logger.warning(f"Failed to clear and stamp: {str(e)}")
```

Ruff (0.14.14)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)